packages/builder/src/components/start/CreateAppModal.svelte

```diff
@@ -22,7 +22,10 @@
   $: validation.check($values)
 
   onMount(async () => {
-    $values.name = resolveAppName(template, $values.name)
+    const defaultName = $auth.user?.firstName
+      ? `${$auth.user.firstName}s app`
```

Comment:
I wish we could put an apostrophe here since we're showing possession.... but app names can't have special characters so we can't. This will also look weird if their name ends in an "s" - e.g. "Chriss app". I think we should account for names ending in "s" and add some extra logic here to not append the second "s".

Reply:
Agreed. I've added accommodation now for user names that end in 'S'.  The tests were updated to reflect this use case.